rotkehlchen/db/filtering.py

```diff
@@ -65,6 +65,26 @@ def prepare(self) -> Tuple[List[str], List[Any]]:
         raise NotImplementedError('prepare should be implemented by subclasses')
 
 
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
+class DBNestedFilter(DBFilter):
+    and_op: bool
+    filters: List[DBFilter]
+
+    def prepare(self) -> Tuple[List[str], List[Any]]:
+        filterstrings = []
+        bindings = []
+        for fil in self.filters:
```

Comment:
Don't make names unreadable by abbreviation

```suggestion
        for filter in self.filters:
```

Reply:
I just copy-pasted it from `DBFilterQuery`. Perhaps `fil` was chosen in the past since `filter` is a reserved python keyword. I'll make it more readable, but perhaps makes sense to use some different word instead of `filter`?

Reply:
yes if it's a reserved keyword use a different one